### PR TITLE
Various cleanups

### PR DESCRIPTION
**Pull request by xen0n**

### CODE REVIEW

File: metadata/.gitignore [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+/md5-cache
+/news
+/herds.xml
+/timestamp
+/timestamp.chk
+/timestamp.x
+/pkg_desc_index
+/projects.xml
+/xml-schema
+/.checksum-test-marker
```
Comment (fosero) [inline metadata/.gitignore]: Can you explain to me why I would need this in an overlay?

Comment (xen0n) [reply metadata/.gitignore]: I simply copy-pasted it from the official repo. Will delete these lines then.

Comment (fosero) [reply metadata/.gitignore]: I think this .gitignore can completely go.. we will never generate these files afaik

Comment (xen0n) [reply metadata/.gitignore]: No, if you do `egencache` on the overlay there would be at least `md5-cache`. Some people (including me) do this for speeding up Portage operations especially when a lot of overlays are activated.

**Comment by fosero** [comment]
Fair enough. Thanks for your work.

**Pull request merged**